geode-core/src/main/java/org/apache/geode/internal/cache/AbstractBucketRegionQueue.java

```diff
@@ -257,11 +273,47 @@ protected void loadEventsFromTempQueue() {
             }
             getInitializationLock().writeLock().unlock();
           }
+          if (regionToDispatchedKeysMap.size() > 0
+              && getPartitionedRegion().getRegionAdvisor() != null) {
+            Set<InternalDistributedMember> recipients =
+                getPartitionedRegion().getRegionAdvisor().adviseDataStore();
+
+            if (recipients.isEmpty()) {
+              return;
+            }
+
+            InternalDistributedSystem ids = getCache().getInternalDistributedSystem();
+            DistributionManager dm = ids.getDistributionManager();
+            dm.retainMembersWithSameOrNewerVersion(recipients, KnownVersion.GEODE_1_15_0);
+
+            if (!recipients.isEmpty()) {
+              ParallelQueueSetPossibleDuplicateMessage pqspdm =
```

Comment:
Could this be named "possibleDuplicateMessage" instead? Variable names like this are often confusing, and we don't need to try and save characters in the source code by using abbreviations.